bin/setup-faxws.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+echo "Setting up the Fax server"
+
+## 3. Clone
+git clone https://dennis_warren@bitbucket.org/openoscar/faxws.git
+
+## 4. Checkout
+git checkout origin/release
+
+# 5. current MySQL pass and username
+echo "Setting Fax WS database passwords... "
+
+read -p "MySQL database username? " username
```

Comment:
What username and password is this for? The main Oscar DB?

Reply:
This would be the username and password for the a user that has create privileges in the main MariaDB database.  FlyWay will create the entire OscarFax database if the MySQL version is 5.7 or newer.
Technically this could be ported from the database credentials that are created during the Oscar and DrugRef installs.